app/app.scss

```diff
@@ -395,3 +395,29 @@ body {
     background-color: #546E7A;
   }
 }
+
+.not-found-icon {
+  text-align: center;
+  margin-top: 130px;
+
+  svg {
+    fill: $colorBlackTextDisabled;
+    margin: 0 auto;
+    height: 40px;
+    width: 40px;
+  }
+}
+
+.not-found-label {
+  text-align: center;
+  font-size: 11px;
```

Comment:
i'm going to merge, but some comments for the future:

* always add line-height multiple of 4px
* all metrics should generally be multiple of 4 (e.g. margin-bottom: 5px below here)